config/application.rb

```diff
@@ -48,8 +48,6 @@ class Application < Rails::Application
 
     config.analytics_tracking_id = ENV["GA_TRACKING_ID"]
     config.i18n.load_path += Dir[Rails.root.join("config/locales/**/**/*.{rb,yml}")]
-    config.filter_parameters << :password
-    config.filter_parameters << :password_confirmation
```

Comment:
Is `password_confirmation` also passed to the parameter_logging. I can only see `passwd` and that's unclear if it includes both.

Reply:
My commit message could be a bit more descriptive. Those params were added to configure Raven (Sentry SDK)
https://github.com/alphagov/datagovuk_find/commit/929fda7059ca4df0cc1514c32b0214805763c63e

We migrated over to Sentry Ruby SDK, which protects sensitive data by default so we no longer need to configure those. The default rails configuration is in `config/initializers/filter_parameter_logging.rb`

I'll move this commit to another PR as suggested because it's not directly related to rails 7.0 upgrade.